frontend/src/components/smart/PvPArenaPreparation.vue

```diff
@@ -295,6 +307,7 @@ export default {
         console.log(this.entryWager);
         console.log('Missing data');
       }
+      this.loading = false;
```

Comment:
With all the loadings and such, if there is a try catch and after that some loading is set to false regardless of the outcome -> I would put it in finally. Maybe not this particular place ofc